client/src/components/Nav.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import { Navbar } from 'react-materialize';
+import { LogIn } from './LogIn';
+import { NavProfile } from './NavProfile';
+import { LogOut } from './LogOut';
+
+
+export class Nav extends Component {
+  constructor(props) {
+    super(props);
+    this.update = this.update.bind(this);
+  }
+
+  isLogged() {
+    return typeof (sessionStorage['Authorization-token']) !== 'undefined' ? true : false
```

Comment:
`typeof (sessionStorage['Authorization-token']) !== 'undefined'` already returns true or false, you don't need here ternary    operator

Reply:
Also I think move this function into scripts.js file, is it ok?

Reply:
no objections from my side